### ISSUE TITLE
Grouping by folder path or topmost folders in the collection

### ISSUE THREAD

**Issue by kik4444** [strawberrymusicplayer/strawberry#381]
**Is your feature request related to a problem? Please describe.**
My music's metadata is not really made to be displayed how I want it from any of the currently available grouping options, so in the meantime I've just manually created saved playlists and dropped all my folders into them so they can recursively display everything in them.
Plus I've also made many symlinks to my favorite tracks which I keep in a separate folder, but they're not usable with the current grouping options because they just appear as duplicates among the real files.
I think this could be fixed with an option to group the collection by topmost folder.

**Describe the solution you'd like**
What about a grouping option based on the collection's top folder? The collection folder I've set simply contains a bunch of subfolders only. And those subfolders either contain my music files directly 1 level down, or several more subfolders deeper. 
I'd like this grouping option to show everything visible from each of the topmost folders in the directory specified in the collection settings. Here's a visual representation (to the best of my abilities). Assume all of the music below has different albums, artists, etc. 

**Raw folder layout from the collection directory:**

Folder1/music1.mp3
Folder1/music2.flac
Folder2/music3.wav
Folder2/Subfolder1/music4.ogg
Folder2/Subfolder2/music5.mp3
Folder3/Subfolder3/music6.flac

**Expected view with this grouping option:**

Folder1/music1.mp3
Folder1/music2.flac
Folder2/music3.wav
Folder2/music4.ogg
Folder2/music5.mp3
Folder3/music6.flac

Basically it shows all of the music inside each subfolder at once, no matter how deep they are, and only divides them by the topmost folder.

**Describe alternatives you've considered**
Perhaps something like Foobar2000's autoplaylist

**Comment by jonaski** [comment]
The collection isn't based folders, it's based on tags in the audio metadata.
There is a file view too, but it's very simplistic. The functionality of the player is mostly based around the collection library.

**Comment by kik4444** [comment]
So are there any plans to to implement this in the future? I've managed to make do with playlists I created manually and dropped folders into them, but I'd like to use the collection tab too, however it's not very convenient when the files aren't displayed in my preferred order and without separating the symlinks (located in different folders) from the real files.

**Comment by jonaski** [comment]
No. It doesn't make sense. I think what you are looking for is a file tree view based player.
